gc/base/standard/Scavenger.cpp

```diff
@@ -1707,6 +1707,33 @@ MM_Scavenger::scavengeObjectSlots(MM_EnvironmentStandard *env, MM_CopyScanCacheS
 	return shouldRemember;
 }
 
+void
+MM_Scavenger::deepScanOutline(MM_EnvironmentStandard *env, omrobjectptr_t objectPtr, uintptr_t gcPriorityScanField1, uintptr_t gcPriorityScanField2, uintptr_t freeListUtilizationLimit){
+
+	void* tempObj = objectPtr;
+	uintptr_t priorityField = gcPriorityScanField1;
+
```

Comment:
it would be nice to add some stats (and report probably as part of TGC paralle for scavenger:
1) how many lists (how many times we entered this method)
2) how many total nodes
3) longest list